spring-boot/admin/src/main/resources/application.yml

```diff
@@ -64,10 +63,6 @@ ygg:
   proxy:
     executable-jar:
       generated.path: /tmp
-      # URL of the "template jar" that CoatRack uses as basis to generate proxies.
-      # During development testing, this should usually point to your locally built coatrack gateway jar
-      # (but without the trailing version number and .jar filename extension, those will be added automatically).
-      template.url: file:///${user.dir}/spring-boot/proxy/target/coatrack-proxy-
```

Comment:
This should not be removed, it should stay configurable.

Reply:
Outdated.